java/arcs/core/data/proto/manifest.proto

```diff
@@ -77,8 +77,9 @@ message ParticleSpecProto {
   repeated ClaimProto claims = 4;
   // Conjunction of all checks.
   repeated CheckProto checks = 5;
-  // Indicates whether the particle is an isolated (non-egress) particle.
-  bool isolated = 6;
```

Comment:
Shouldn't we mark this as `[deprecated = true]` instead of making it reserved? Usually `reserved` means unused currently and may be used in future?